packages/analytics-js-integrations/__tests__/integrations/GA4/browser.test.js

```diff
@@ -108,15 +104,20 @@ describe('Google Analytics 4 Group events tests', () => {
       }
     });
   });
-})
+});
 
 describe('Google Analytics 4 Identify events tests with no piiPropertiesToIgnore', () => {
   let ga4;
   // Config 1 : default piiPropertiesToIgnore value
   beforeEach(() => {
     ga4 = new GA4(
-      { measurementId: 'G-123456', extendPageViewParams: true, capturePageView: 'rs', piiPropertiesToIgnore: [{ piiProperty: '' }] },
-      { getUserId: () => '1234', getUserTraits: () => { } },
+      {
```

Comment:
input.config can be used?

Reply:
we are taking configs from test case because for each test config is changing in page call, for for rest of the call config is same so no need to add it in tests payload